models/articles.model.js

```diff
@@ -0,0 +1,18 @@
+const db = require("../db/connection")
+
+
+exports.selectArticlesById = (article_id) => {
+    return db
+    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .then((result) => {
+        if(result.rowCount < 1){
+            return Promise.reject({status: 404, msg: "Not Found!"})
+        }
+       return result.rows[0]
+    })
+    .catch((err)=> {
+        console.error(err)
+        throw err
+    })
```

Comment:
Do you need this catch block? 👀 
Also keep in mind that you `throw` the error, it won't reach your middleware.